test/Error/rlexe.xml

```diff
@@ -30,6 +30,8 @@
     <default>
       <files>stack.js</files>
       <baseline>stack.baseline</baseline>
+      <!-- xplat-todo: Understand why this test is failing on OS X -->
+      <tags>exclude_mac</tags>
```

Comment:
That object inline test fail also first for me. Can we have some env. detail on osx CI? we might also need to change test timeouts